assets/styles/layouts/_Blog.scss

```diff
@@ -52,6 +52,96 @@
 			z-index: 2;
 		}
 
+		&__navigation {
```

Comment:
Please try to use same class format as we are used to, ie &__navigation--nav, &__navigation--description, &__navigation--item etc., not just „.nav“, „.menu-item“